jamira-cli/src/main/java/org/tomitribe/jamira/cli/Jamira.java

```diff
@@ -15,31 +15,31 @@
  */
 package org.tomitribe.jamira.cli;
 
-import org.tomitribe.util.dir.Mkdir;
-
 import java.io.File;
 import java.util.stream.Stream;
 
+import org.tomitribe.util.dir.Mkdir;
+
 public interface Jamira extends org.tomitribe.util.dir.Dir {
 
     @Mkdir
     Cache cache(final String name);
 
-    default Cache cache(final Account account) {
-        return cache(account.getName());
+    public static Cache cache(Jamira jamira, final Account account) {
+        return jamira.cache(account.getName());
     }
 
-    default Account account() {
-        return account("default");
+    public static Account account(Jamira jamira) {
+        return account(jamira, "default");
     }
 
-    default Account account(final String name) {
-        final File file = file(name + ".properties");
+    public static Account account(Jamira jamira, final String name) {
+        final File file = jamira.file(name + ".properties");
```

Comment:
Yeah, I'll need to take another look into this one; the default method on a proxy-ied class gave me some exception, it might even be a bug in GraalVM.